include/roboteam_ai/world/Field.h

```diff
@@ -1,58 +1,264 @@
+    TOP_LINE, // The field line with the highest y-coordinate which goes from the left side to the right side of the field.
+    BOTTOM_LINE, // The field line with the lowest y-coordinate which goes from the left side to the right side of the field.
+    LEFT_LINE, // The field line left from our goal (our goal is always adjacent to this line).
+    RIGHT_LINE, // The field line right from the opponents goal (their goal is always adjacent to this line)
+    /* The line that seperates our side from the field and the opponent side of the field (this line moves in the y
+     * direction of the field, i.e. the width of the field) */
+    HALF_LINE,
+    /* The line that start from the middle point of our goal and ends at the middle point of the opponents goal (this
+     * line moves in the x direction of the field, i.e. the length of the field) */
+    CENTER_LINE,
+    LEFT_PENALTY_LINE, // The line parallel to the left line and our goal. This line marks our goal area.
+    RIGHT_PENALTY_LINE, // The line parallel to the right line and the opponents goal. This line marks their goal area.
+    /* The line closest (of all our goal area lines) to the top line and parallel to the top line. This line marks our
+     * goal area. */
+    TOP_LEFT_PENALTY_STRETCH,
+    /* The line closest (of all our goal area lines) to the bottom line and parallel to the bottom line. This line marks
+     * our goal area. */
+    BOTTOM_LEFT_PENALTY_STRETCH,
+    /* The line closest (of all the opponents goal area lines) to the top line and parallel to the top line. This line
+     * marks the opponents goal area. */
+    TOP_RIGHT_PENALTY_STRETCH,
+    /* The line closest (of all the opponents goal area lines) to the bottom line and parallel to the bottom line. This
+     * line marks the opponents goal area. */
+    BOTTOM_RIGHT_PENALTY_STRETCH,
+
+    // Enum item used to indicate the number of Field Line Names (make sure that this always is the last item of this enum)
+    NUMBER_FIELD_LINE_NAMES
+};
+
+enum FieldVectorName {
+    OUR_GOAL_CENTER, // The middle point of our goal (this point is on the left line).
+    THEIR_GOAL_CENTER, // The middle point of the opponents goal (this point is on the right line).
+    LEFT_PENALTY_POINT, // The penalty point from which penalties are made towards our goal.
+    RIGHT_PENALTY_POINT, // The penalty point from which penalties are made towards the opponents goal.
+    OUR_BOTTOM_GOAL_SIDE, // The bottom most point of our goal (this point is on the left line).
+    OUR_TOP_GOAL_SIDE, // The top most point of our goal (this point is on the left line).
+    THEIR_BOTTOM_GOAL_SIDE, // The bottom most point of the opponents goal (this point is on the right line).
+    THEIR_TOP_GOAL_SIDE, // The top most point of the opponents goal (this point is on the right line).
+
+    // Enum item used to indicate the number of Field Vector Names (make sure that this always is the last item of this enum)
+    NUMBER_FIELD_VECTOR_NAMES
```

Comment:
No! Why are you doing this!? By evading the getters and setters, which were fine, you now have to go to these kinds of solutions? These things are really indications that you are overcomplicating things.